frontend/public/components/monitoring/_monitoring.scss

```diff
@@ -336,6 +336,11 @@ $monitoring-line-height: 18px;
   display: flex;
 }
 
+.query-browser__reduced-resolution.pf-c-alert{
+  --pf-c-alert--GridTemplateAreas: "icon title action";
```

Comment:
Is the `--pf-c-alert--GridTemplateAreas: "icon title action"` needed? I tried it without and it looked like we still get the desired result, but maybe I missed something?

Reply:
You are right, is not needed anymore was part of the previous solution. 🙇🏽